python-checks/src/main/java/org/sonar/python/checks/cdk/AbstractCdkResourceCheck.java

```diff
@@ -49,8 +50,8 @@ public void initialize(SubscriptionCheck.Context context) {
 
   protected void visitNode(SubscriptionContext ctx) {
     CallExpression node = (CallExpression) ctx.syntaxNode();
-    Optional.ofNullable(node.calleeSymbol())
-      .map(Symbol::fullyQualifiedName)
+    Optional.of(node)
+      .map(TreeUtils::fullyQualifiedNameFromExpression)
```

Comment:
it looks like the previous implementation still works, i.e.
 ```java
Optional.ofNullable(node.calleeSymbol())
      .map(Symbol::fullyQualifiedName)
```
So I would suggest to use the old code, and to revert the changes done inside `TreeUtils::fullyQualifiedNameFromExpression` , sorry! :D

Reply:
My bad, this one was lost in the many change that happened regarding FQN, I didn't saw that I did not restore the code properly.